src/Modules/Manage/Dnn.PersonaBar.Sites/Sites.Web/src/components/CreatePortal/index.jsx

```diff
@@ -31,19 +31,16 @@ const emptyNewPortal = {
     PasswordConfirm: ""
 };
 function extractDomain(url) {
-    let domain;
+    let domain, port;
     //find & remove protocol (http, ftp, etc.) and get domain
     if (url.indexOf("://") > -1) {
         domain = url.split("/")[2];
     }
     else {
         domain = url.split("/")[0];
     }
-
-    //find & remove port number
-    domain = domain.split(":")[0];
-
-    return domain;
+    [domain, port] = domain.split(":");
+    return port != undefined && port != 80 ? `${domain}:${port}` : domain;
```

Comment:
Seeing how you handle `80` as special case, same should be done with port `443` when the protocol is https IMO